atom/common/native_mate_converters/blink_converter.cc

```diff
@@ -159,10 +159,17 @@ bool Converter<blink::WebKeyboardEvent>::FromV8(
   if (!ConvertFromV8(isolate, val, static_cast<blink::WebInputEvent*>(out)))
     return false;
   base::char16 code;
-  if (!dict.Get("keyCode", &code))
-    return false;
+  std::string identifier;
   bool shifted = false;
-  out->windowsKeyCode = atom::KeyboardCodeFromCharCode(code, &shifted);
+
+  if (dict.Get("keyCode", &code))
+    out->windowsKeyCode = atom::KeyboardCodeFromCharCode(code, &shifted);
+  else if (dict.Get("keyIdentifier", &identifier))
```

Comment:
I think we can use `keyCode` for both the char and identifier, if the input string has length 1 then convert it to `code `, otherwise convert it to `identifier`, changing the `keyIdentifier` in this line to `keyCode` can do the work automatically.